docs/UserGuide.md

```diff
@@ -63,109 +63,137 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
 </div>
 
-### Viewing help : `help`
+### **Viewing Help**
 
-Shows a message explaning how to access the help page.
+Shows a message explaining how to access the help page.
+
+**Command Format:**  
+`help`
+
+- Provides help information.
 
 ![help message](images/helpMessage.png)
 
-Format: `help`
+---
 
+### **Add Contact**
 
-### Adding a person: `add`
+Adds a new contact with details such as name, phone number, physical address, birthday, email, social media handles, and remarks.
 
-Adds a person to the address book.
+**Command Format:**  
+`add n/<Full Name> p/<Phone Number> a/<Address> [b/<Birthday>] e/<Email> [r/<Remark>] [t/<Tag>]`
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+- `n/` Full name (mandatory)
+- `p/` Phone number (mandatory)
+- `a/` Address (mandatory)
+- `b/` Birthday (optional)
+- `e/` Email (optional)
+- `r/` Remark (optional)
+- `t/` Tag (optional)
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
-Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+**Examples:**
+- `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
+- `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
```

Comment:
Can perhaps add more examples ie.  for birthday etc since they might not know the format